source/features/ci-link.tsx

```diff
@@ -1,35 +1,74 @@
 import './ci-link.css';
 import select from 'select-dom';
 import onetime from 'onetime';
+import {observe} from 'selector-observer';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 import fetchDom from '../helpers/fetch-dom';
-import {buildRepoURL} from '../github-helpers';
+import {buildRepoURL, getConversationNumber} from '../github-helpers';
+
+const deinit: VoidFunction[] = [];
 
 // Look for the CI icon in the latest 2 days of commits #2990
-const getIcon = onetime(async () => fetchDom(
+const getRepoIcon = onetime(async () => fetchDom<HTMLElement>(
 	buildRepoURL('commits'), [
 		'.commit-group:nth-of-type(-n+2) .commit-build-statuses', // Pre "Repository refresh" layout
 		'.TimelineItem--condensed:nth-of-type(-n+2) .commit-build-statuses'
 	].join()
 ));
 
-async function init(): Promise<false | void> {
-	const icon = await getIcon() as HTMLElement | undefined;
+const getPRIcon = onetime(async () => select(
+	'.js-commits-list-item:last-of-type .commit-build-statuses',
+	pageDetect.isPRCommitList() ? document : await fetchDom(
+		buildRepoURL('pull', getConversationNumber()!, 'commits')
+	)
+));
+
+async function initRepo(): Promise<false | void> {
+	const icon = await getRepoIcon();
 	if (!icon) {
 		return false;
 	}
 
 	icon.classList.add('rgh-ci-link');
-	if (onetime.callCount(getIcon) > 1) {
+	if (onetime.callCount(getRepoIcon) > 1) {
 		icon.style.animation = 'none';
 	}
 
 	// Append to title (aware of forks and private repos)
 	select('[itemprop="name"]')!.parentElement!.append(icon);
 }
 
+async function initPR(): Promise<false | void> {
+	const icon = await getPRIcon() as HTMLElement | undefined;
```

Comment:
I can't inline this type in `getPRIcon` because eslint thinks `select` returns `HTMLElement` by default while typescript sees it `Element`.

Reply:
Must be a TS bug. Fixed by using `<HTMLElement>` on the select itself